tests/Console/ConfigurationResolverTest.php

```diff
@@ -443,7 +443,11 @@ public static function provideRejectInvalidPathCases(): iterable
     public function testResolvePathWithFileThatIsExcludedDirectlyOverridePathMode(): void
     {
         $config = new Config();
-        $config->getFinder()
+        $finder = $config->getFinder();
+
+        assert($finder instanceof Finder); // Config::getFinder() ensures only `iterable`
```

Comment:
Why `assert()` instead of `self::assertInstanceOf()` ?

Reply:
this assertion was not added because of test case assertions needs, but because of PHPStan 

we use `assert` when we want to hint PHPStan on the type.